Create NIBM/2022-current.md

dileepadev-old opened the pull request:
Directory for the NIBM FOSS Community @nibmfossc.

- Dileepa Bandara @dileepabandara  
  contact.dileepabandara@gmail.com  
  Club Lead, NIBM FOSS Community @nibmfossc

dileepadev-old committed:
Create NIBM/2022-current.md

dhanushkac commented on cc/NIBM/2022-current.md:
```diff
@@ -0,0 +1,62 @@
+- The executive committee will then get in touch with the applicant and add them as a new member of the NIBM Computing Society.
+- Members who are willing to be a part of the FOSS Community will be added to the NIBM FOSS Community.
+
+Please contact the following executives for further details:  
```
Below part is not relavant.

dhanushkac commented on cc/NIBM/2022-current.md:
```diff
@@ -0,0 +1,62 @@
+- Master in Charge  
+  Mr. Thisara Weerasinghe  
+  thisara@nibm.lk  
+  [LinkedIn](https://lk.linkedin.com/in/thisara-weerasinghe-687b55a3)  
```
This is for the tracking purpose. Just the name is enough.

dileepadev-old committed:
Update NIBM/2022-current.md

Resolved:  
- https://github.com/fosslk/Campus-Club/pull/50#discussion_r1046060656
- https://github.com/fosslk/Campus-Club/pull/50#discussion_r1046061980

dhanushkac reviewed (approved):
LGTM

dileepadev-old commented:
Thanks @sudarakas and @dhanushkac 😊🎉
